frontend/src/components/molecules/UsersList/UserList.tsx

```diff
@@ -0,0 +1,22 @@
+import { isStartsWith } from 'helpers';
+import { IUsers } from './UserList.model';
+import { StyledList } from './UserList.styled';
+import User from './User';
+
+const UserList = ({ users, searchValue }: IUsers): JSX.Element => {
+  return (
```

Comment:
We are using `users` only in this component. Please retrieve them inside this component instead of retrieving them on the parent side and just drill to child